tests/plugins/colorbutton/manual/select.html

```diff
@@ -0,0 +1,74 @@
+<style>
+#selected-color {
+	margin-top: 10px;
+	width: 100px;
+	height: 20px;
+	border: 1px solid black;
+}
+</style>
+<div id="editor"></div>
+<select id="choose-color"></select>
+<div id="selected-color"></span>
+
+<script>
+	var editor = CKEDITOR.replace( 'editor' );
+
+	editor.once( 'instanceReady', overwriteUIOpen( 'TextColor' ) );
+	editor.once( 'instanceReady', overwriteUIOpen( 'BGColor' ) );
+	editor.once( 'instanceReady', initializeColorSelector );
```

Comment:
Can't it be joined into one listener?

Reply:
It can, I just like to pass callbacks by reference, so it's more readable. But I'm fine for merging it into a single listener.